class LoginError(Exception):
    pass


class AccountNotFoundError(Exception):
    pass
